deploy-agent/deployd/client/client.py

```diff
@@ -175,8 +175,8 @@ def send_reports(self, env_reports=None):
             if self._read_host_info():
                 reports = [status.report for status in env_reports.values()]
                 for report in reports:
-                    if isinstance(report.errorMessage, str):
-                        report.errorMessage = unicode(report.errorMessage, "utf-8")
+                    if isinstance(report.errorMessage, bytes):
```

Comment:
will this work for both py2 and py3?  surprising to see the type "str" and "bytes" are the same

Reply:
yes, under PY2 `bytes==str`